src/main/java/org/opentripplanner/middleware/models/MonitoredTrip.java

```diff
@@ -29,6 +32,12 @@
      */
     public String tripTime;
 
+    /**
+     * Tracks information during the active monitoring of the trip (e.g., last alerts encountered, last time a check was
+     * made, etc.).
+     */
+    public JourneyState journeyState = new JourneyState();
```

Comment:
Should the JourneyState be stored in it's own collection instead of being nested here? I'm pretty sure we don't want to expose the information to the end user or make it possible to update from a webservice.

Reply:
Perhaps. We definitely don't want to share this with the end user.